site-src/guides/migrating-from-ingress.md

```diff
@@ -0,0 +1,261 @@
+
+The Gateway API supports all the features of the Ingress resources and many features that are only available through annotations. As a result, the Gateway API is more portable than the Ingress API. Additionally, as the next section will show, you will not need to use any annotations at all, which addresses the awkwardness problem. 
+
+### Approach to Extensibility
+
+The Ingress API has two extensions points:
+
+* Annotations on the Ingress resource (described in the previous section).
+* [Resource backends](https://kubernetes.io/docs/concepts/services-networking/ingress/#resource-backend), which is the ability to specify a backend other than a Service.
+
+The Gateway API is feature-rich compared with the Ingress API. However, to configure some advanced features like authentication or common but non-portable across data planes features like connection timeouts and health checks, you will need to rely on the extensions of the Gateway API.
+
+The Gateway API has the following primary extension points:
+
+* *External references.* A feature (field) of a Gateway API resource can reference a custom resource (CRD) specific to the Gateway implementation that configures that feature. For example:
+    * [HTTPRouteFilter](/references/spec/#gateway.networking.k8s.io/v1beta1.HTTPRouteFilter) can reference an external resource via the `extensionRef` field, thus configuring an implementation-specific filter.
+    * [BackendObjectReference](/references/spec/#gateway.networking.k8s.io/v1beta1.BackendObjectReference) supports resources other than Services.
+    * [SecretObjectReference](/references/spec/#gateway.networking.k8s.io/v1beta1.SecretObjectReference) supports resources other than Secrets.
+* *Custom implementations*. For some features, it is left up to an implementation to define how to support them. Those features correspond to the implementation-specific (custom)  [conformance level](/concepts/conformance/#2-support-levels). For example:
+    * The `RegularExpression` type of the [HTTPPathMatch](/references/spec/#gateway.networking.k8s.io/v1beta1.SecretObjectReference).
+* *Policies.* A Gateway implementation can define custom resources (CRDs) called Policies for exposing data plane features like authentication. The Gateway API does not prescribe the details of those resources. However, it prescribes a standard UX. See the [Policy attachment guide](/references/policy-attachment/) for more details. In contrast with the *external references* above, a Gateway API resource does not reference a Policy. Instead, a Policy must reference a Gateway API resource.
+
+The extension points do not include annotations on the Gateway API resources. This approach is strongly discouraged for Gateway implementations.
+
+## Mapping Ingress API features to Gateway API Features
+
+This section will map the Ingress API features to the corresponding Gateway API features. It will cover three major areas:
+
+* Entry points.
+* TLS termination.
+* Routing rules.
```

Comment:
Tiny nit: I don't think these need to end with `.` because they're not sentences.